vmdb/app/models/ems_refresh/parsers/kubernetes.rb

```diff
@@ -58,6 +58,7 @@ def parse_service(service)
         :namespace        => service.metadata.instance_values["table"][:namespace],
         :session_affinity => service.spec.sessionAffinity
       )
+      new_result[:labels] = parse_labels(service)
```

Comment:
This can just be a key in the prior merge statement (I generally leave a blank line between properties and relationships for readability).